papaparse.js

```diff
@@ -522,6 +526,7 @@ License: MIT
 				this._baseIndex = lastIndex;
 			}
 
+			 this._currentByteCursor += this._partialLine ? this.calculateTotalByteByString(aggregate) - this.calculateTotalByteByString(this._partialLine) : this.calculateTotalByteByString(aggregate)
```

Comment:
out of curiousity, why we don't handle this inside Parser.parse?

Reply:
This is not an optimal solution indeed, will make changes later today to put logic inside parse() function.